.ci/Jenkinsfile

```diff
@@ -482,6 +482,7 @@ def generateStepForWindows(Map params = [:]){
           deleteDir()
           unstash 'source'
           dir(BASE_DIR) {
+            bat label: 'Ping linux worker', script: "ping -n 3 ${linuxIp}"
```

Comment:
For debugging purposes, let's add a line with the details regarding what's the IP address that the windows CI Worker will connect to. If it's not reachable then it should fail earlier